docs/integrations/destinations/s3.md

```diff
@@ -377,12 +377,29 @@ Like most of the other Airbyte destination connectors, usually the output has th
 an emission timestamp, and the data blob. With the CSV output, it is possible to normalize
 \(flatten\) the data blob to multiple columns.
 
-| Column                | Condition                                                                                         | Description                                                              |
-| :-------------------- | :------------------------------------------------------------------------------------------------ | :----------------------------------------------------------------------- |
-| `_airbyte_ab_id`      | Always exists                                                                                     | A uuid assigned by Airbyte to each processed record.                     |
-| `_airbyte_emitted_at` | Always exists.                                                                                    | A timestamp representing when the event was pulled from the data source. |
-| `_airbyte_data`       | When no normalization \(flattening\) is needed, all data reside under this column as a json blob. |                                                                          |
-| root level fields     | When root level normalization \(flattening\) is selected, the root level fields are expanded.     |                                                                          |
+| Column                   | Condition                                                                                         | Description                                                              |
+|:-------------------------|:--------------------------------------------------------------------------------------------------|:-------------------------------------------------------------------------|
+| `_airbyte_raw_id`        | Always exists                                                                                     | A uuid assigned by Airbyte to each processed record.                     |
+| `_airbyte_extracted_at`  | Always exists.                                                                                    | A timestamp representing when the event was pulled from the data source. |
```

Comment:
[nit]

```suggestion
| `_airbyte_extracted_at`  | Always exists.                                                                                    | A timestamp representing when the event was loaded from the data source. |
```

Might as well use the term of art (Extract and Load)

Reply:
fixed